src/test/scala/com/cognite/spark/datasource/EventsRelationTest.scala

```diff
@@ -161,6 +162,54 @@ class EventsRelationTest extends FlatSpec with Matchers with SparkTest {
     assert(eventsRead == 1300)
   }
 
+  it should "handle pusdown filters on eventIds" taggedAs WriteTest in {
+    val metricsPrefix = "pushdown.filters.id"
+    val df = spark.read
+      .format("com.cognite.spark.datasource")
+      .option("apiKey", writeApiKey.apiKey)
+      .option("type", "events")
+      .option("collectMetrics", "true")
+      .option("metricsPrefix", metricsPrefix)
+      .load()
+      .where("id = 370545839260513")
+    assert(df.count == 1)
+    val eventsRead = getNumberOfRowsRead(metricsPrefix, "events")
+    assert(eventsRead == 1)
+  }
+
+  it should "handle pusdown filters on many eventIds" taggedAs WriteTest in {
+    val metricsPrefix = "pushdown.filters.ids"
+    val df = spark.read
+      .format("com.cognite.spark.datasource")
+      .option("apiKey", writeApiKey.apiKey)
+      .option("type", "events")
+      .option("collectMetrics", "true")
+      .option("metricsPrefix", metricsPrefix)
+      .load()
+      .where("id In(607444033860, 3965637099169, 10477877031034, 17515837146970, 19928788984614, 21850891340773)")
```

Comment:
Add a test using both `id in` and `externalId in`. I think that for now, it's ok to not support externalId at all.

Reply:
How can I find an externalId?